index.src.html

```diff
@@ -1412,19 +1198,12 @@ <h4 id="body-extraction" algorithm>
       That is, {{Credential}} objects for `https://example.com` would not be returned directly to
       `https://www.example.com`, but could be offered to the user via the chooser.
 
-  {{PasswordCredential}}s further mitigate the risk of data leakage by never exposing the
-  {{[[password]]}} slot directly to a page's JavaScript, but only allowing its submission to a
-  same-origin server-side endpoint via {{fetch()}}.
-
-  ### Same-origin Leakage ### {#passwords-same-origin-leakage}
-
-  Cross-site scripting attacks could make it possible to exploit {{PasswordCredential}}'s
-  integration with {{fetch()}} in order to leak credential information via a vulnerable
-  same-origin endpoint. To mitigate this kind of risk, authors SHOULD set a reasonable Content
-  Security Policy which restricts the kinds of content which can execute in their sites' context
-  [[CSP]]. Moreover, authors should ensure that both <a>`script-src`</a> and <a>`object-src`</a>
+  Cross-site scripting attacks could leak credential information. To mitigate this kind of risk,
+  authors should ensure that both <a>`script-src`</a> and <a>`object-src`</a>
   directives are set in any policy they specify in order to ensure that only trusted script is
-  executed on a page with access to a user's credentials.
+  executed on a page with access to a user's credentials. If you are populating
+  {{HTMLFormElement|HTMLFormElements}}, also <a>`form-action`</a> directives
```

Comment:
Nit: `<{form}> elements` (`HTMLFormElement` is the IDL representation of the HTML element)